Sources/POI/OAPOIUIFilter.mm

```diff
@@ -1071,12 +1072,20 @@ - (void) setTypeToAccept:(OAPOICategory *)poiCategory b:(BOOL)b
 
 - (NSString *) getIconId
 {
+    NSString *iconName;
     if ([filterId hasPrefix:STD_PREFIX])
-        return standardIconId;
+        iconName = standardIconId;
     else if ([filterId hasPrefix:USER_PREFIX])
-        return [[filterId substringFromIndex:USER_PREFIX.length] lowerCase];
-
-    return filterId;
+        iconName = [[filterId substringFromIndex:USER_PREFIX.length] lowerCase];
+    if ([OASvgHelper hasMapImageNamed:iconName])
```

Comment:
Not good idea to check if file exists at this method. Whole fix should be discussed.